manuscript/setup/dev.md

```diff
@@ -74,6 +74,7 @@ gcloud container get-server-config --region us-east1
 # Replace `[...]` with a valid master version from the previous output.
 export K8S_VERSION=[...]
 
+# VIKTOR - for me this makes a 3-node cluster... but I see that '--num-nodes 1' flag
```

Comment:
GKE is "special". When using zonal clusters (all nodes in the same zone/datacenter), the `--num-nodes` represents the number of nodes (as expected). However, when using regional clusters (clusters spanning multiple zones/datacenters), the `--num-nodes` argument specifies the number of nodes in each zone. The region used in this example has three zones, hence the total number of nodes is multiplied by three. It's confusing...